solutions/fully-configurable/variables.tf

```diff
@@ -556,3 +556,43 @@ variable "skip_ocp_secrets_manager_iam_auth_policy" {
   description = "To skip creating auth policy that allows OCP cluster 'Manager' role access in the existing Secrets Manager instance for managing ingress certificates."
   default     = false
 }
+
+##############################################################
+# Kube Audit
+##############################################################
+
+variable "enable_kube_audit" {
+  type        = bool
+  description = "Kubernetes audit logging provides a chronological record of operations performed on the cluster, including by users, administrators, and system components. It is useful for compliance, and security monitoring. Set true to enable kube audit by default. [Learn more](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/)"
```

Comment:
This is not the correct link to link to. We should link to IBM Cloud docs: https://cloud.ibm.com/docs/containers?topic=containers-health-audit#audit-api-server